core/controllers/feedback_test.py

```diff
@@ -195,47 +195,50 @@ def test_post_message_to_existing_thread(self):
         response = self.testapp.get('/create/%s' % self.EXP_ID)
         csrf_token = self.get_csrf_token_from_response(response)
 
-        # First, create a thread.
-        self.post_json(
-            '%s/%s' % (feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID), {
-                'state_name': None,
-                'subject': u'New Thread ¡unicode!',
-                'text': u'Message 0 ¡unicode!',
-            }, csrf_token)
-
-        # Then, get the thread id.
-        response_dict = self.get_json(
-            '%s/%s' % (feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID))
-        threadlist = response_dict['threads']
-        self.assertEqual(len(threadlist), 1)
-        thread_id = threadlist[0]['thread_id']
-
-        # Then, create a new message in that thread.
-        thread_url = '%s/%s' % (feconf.FEEDBACK_THREAD_URL_PREFIX, thread_id)
-        self.post_json(
-            thread_url, {
-                'updated_status': None,
-                'updated_subject': None,
-                'text': 'Message 1'
-            }, csrf_token)
+        with self.swap(feconf, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', True):
+            # First, create a thread.
+            self.post_json(
+                '%s/%s' % (
+                    feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID), {
+                        'state_name': None,
+                        'subject': u'New Thread ¡unicode!',
+                        'text': u'Message 0 ¡unicode!',
+                    }, csrf_token)
+
+            # Then, get the thread id.
+            response_dict = self.get_json(
+                '%s/%s' % (feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID))
+            threadlist = response_dict['threads']
+            self.assertEqual(len(threadlist), 1)
+            thread_id = threadlist[0]['thread_id']
+
+            # Then, create a new message in that thread.
+            thread_url = '%s/%s' % (
+                feconf.FEEDBACK_THREAD_URL_PREFIX, thread_id)
+            self.post_json(
+                thread_url, {
+                    'updated_status': None,
+                    'updated_subject': None,
+                    'text': 'Message 1'
+                }, csrf_token)
 
-        # The resulting thread should contain two messages.
-        response_dict = self.get_json(thread_url)
+            # The resulting thread should contain two messages.
```

Comment:
extra space?

Reply:
This is actually an indent to be within the "with self.swap(....)" block.